calcom/cal.com
# feat: Use Stripe paymentElement with multiple payment methods

## What does this PR do?

- Changes payment layout to accordion
- Adds all permitted options for immediate transactions (scheduled/delayed transactions are still card only).

- Fixes #18049 (GitHub issue number)
- Fixes CAL-4858 (Linear issue number - should be visible at the bottom of the GitHub issue description)
(https://www.loom.com/share/024bbb7cd17947edb33f773ee61fa4d8?sid=8d794599-0c88-4b79-b24e-9cdaa6778d89)

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1. On the main Stripe account that Cal.com uses, be sure to enable [payment methods for connected accounts](https://docs.stripe.com/connect/payment-methods)
2. Add the stripe account and connect whatever test account is desired.
3. Modify the connected test account for whatever payment methods you wish to view. 
4. Book a paid meeting to view the payment methods.

- Are there environment variables that should be set?
- NEXT_PUBLIC_STRIPE_PUBLIC_KEY, STRIPE_PRIVATE_KEY, and STRIPE_CLIENT_ID must be set to the account calcom intends to be using.
- What are the minimal test data to have?
- n/a
- What is expected (happy path) to have (input and output)?
- on paid bookings, multiple options may appear.
- Any other important info that could help to test that PR

/claim #18049
